src/BuildScriptGenerator/PhpVersions.cs

```diff
@@ -39,6 +39,6 @@ public static class PhpVersions
         public const string Php56Version = "5.6.40";
         public const string Php56Keys = "0BD78B5F97500D450838F95DFE857D9A90D90EC1 6E4F6AB321FDC07F2C332E3AC2BF0BC433CFC8B3";
         public const string Php56TarSha256 = "1369a51eee3995d7fbd1c5342e5cc917760e276d561595b6052b21ace2656d1c";
-        public static readonly List<string> RuntimeVersions = new List<string> { "5.6", "7.0", "7.2", "7.3", "7.4", "8.0", "8.1", "8.2" };
+        public static readonly List<string> RuntimeVersions = new List<string> { "5.6", "7.0", "7.2", "7.3", "7.4", "8.0", "8.1" };
```

Comment:
@william-msft Have you ever seen this in the past when running the automation? Looks like we have an inconsistency somewhere and it tries to remove the latest 8.2 version.

Reply:
This happened because `8.2` was accidentally removed in the constants.yaml previously (note 8.2 is missing): https://github.com/microsoft/Oryx/pull/1812/files#diff-47c28d7a6c8135707f46b624b5913e35beea6dfbe7a8be2db7efefde606eba59R210

So when Automation runs, it will distribute all constants.yaml changes throughout the Oryx repo. Automation ran as expected. But I'll remedy this change after I get done with the ICM.

Reply:
I submitted a PR for this: https://github.com/microsoft/Oryx/pull/1819/files

Reply:
Ah great. Sorry I missed the email, I just created a PR for the same change. I'll close mine.

Reply:
This has been resolved and Nightly tests are all passing.